encoders/image/CustomKerasImageEncoder/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jinaai/jina
+
+# setup the workspace
+COPY . /workspace
+WORKDIR /workspace
+
+# install the third-party requirements
+RUN apt-get update && apt-get install --no-install-recommends -y git curl && \
+    pip install -r requirements.txt
+
+# RUN pip install -r requirements.txt
+COPY . /
```

Comment:
This copy is not needed here, it was already copied

Reply:
done

Reply:
Still here

Reply:
done, removed copy